common/defs/bitcoin/etp.json

```diff
@@ -0,0 +1,46 @@
+{
+  "coin_name": "MetaverseETP",
+  "coin_shortcut": "ETP",
+  "coin_label": "Metaverse ETP",
+  "website": "https://mvs.org",
+  "github": "https://github.com/mvs-org/metaverse",
+  "maintainer": "Sven Mutzl <sven@mutzl.org>",
+  "curve_name": "secp256k1",
+  "address_type": 50,
+  "address_type_p2sh": 5,
+  "maxfee_kb": 2000000, 
+  "minfee_kb": 100,
+  "signed_message_header": "Metaverse Signed Message:\n",
+  "hash_genesis_block": "b81848ef9ae86e84c3da26564bc6ab3a79efc628239d11471ab5cd25c0684c2d",
+  "xprv_magic": 76066276,
+  "xpub_magic": 76067358,     
+  "xpub_magic_segwit_p2sh": null,
+  "xpub_magic_segwit_native": null,
+  "bech32_prefix": null,
+  "cashaddr_prefix": null,
+  "slip44": 2302,
+  "segwit": false,
+  "decred": false,
+  "fork_id": null,
+  "force_bip143": false,
+  "bip115": false,
+  "default_fee_b": {
+    "Low": 10,
+    "Economy": 70,
+    "Normal": 140,
+    "High": 200
+  },
+  "dust_limit": 546,
+  "blocktime_seconds": 600,
+  "uri_prefix": "etp",
+  "min_address_length": 27,
+  "max_address_length": 34,
+  "bitcore": [],
+  "blockbook": [
+    "https://explorer.mvs.org/api"
```

Comment:
This is not a blockbook. Remove this.

Reply:
@prusnak Oh! I see. Thanks! I've removed it and updated the icon now it's passing the format check. However, I'm getting an error saying `Files src/trezorlib/coins.json and /tmp/tmp.U8aWo9Dv2W differ`. How do I fix this? Also, should I modify other files such as support.json?